Source/Core/VideoBackends/Software/Rasterizer.h

```diff
@@ -29,6 +29,7 @@ struct RasterBlockPixel
 {
   float InvW;
   float Uv[8][2];
+  bool mask;
```

Comment:
I don't like the way that basically all uses of this need to be negated.  I also don't really like the name; it seems slightly ambiguous to me.  Maybe `should_draw` could be used instead (along with getting rid of the negations?)